### PR TITLE
Feat/add advanced navigation

### PR DESCRIPTION
**Pull request by afonsocarlos**
Implements #194 

New features added:
- [x] Move to parent directory without closing
- [x] Navigate through sibling directory/files
- [x] Go to the first/last sibling of the current node

### CODE REVIEW

File: lua/nvim-tree/lib.lua [hunk 1]
```diff
@@ -78,6 +78,31 @@ local function get_node_at_line(line)
   return iter
 end
 
+local function get_line_from_node(node, find_parent)
+  local sep = package.config:sub(1,1)
+  local node_path = node.absolute_path
+
+  if find_parent then
+    node_path = node.absolute_path:match("(.*)"..sep)
```
Comment (kyazdani42) [inline lua/nvim-tree/lib.lua:86]: there are utils to handle the path separator in `utils.lua`

**Review by kyazdani42 — CHANGES_REQUESTED** [review]
Thanks for the PR, but please use existing utils code before i properly review it :)

File: lua/nvim-tree/config.lua [hunk 2]
```diff
@@ -67,9 +67,14 @@ function M.get_bindings()
     ["<C-v>"]          = M.nvim_tree_callback("vsplit"),
     ["<C-x>"]          = M.nvim_tree_callback("split"),
     ["<C-t>"]          = M.nvim_tree_callback("tabnew"),
+    ["<C-k>"]          = M.nvim_tree_callback("prev_sibling"),
+    ["<C-j>"]          = M.nvim_tree_callback("next_sibling"),
```
Comment (kyazdani42) [inline lua/nvim-tree/config.lua:71]: hum i use those two for moving between up and down windows and i think many people would too. maybe something like `<` or `>` ?

**Review by kyazdani42 — CHANGES_REQUESTED** [review]
this little change and i'll merge this :)

File: lua/nvim-tree/config.lua (hunk 2 above)
Comment (afonsocarlos) [reply lua/nvim-tree/config.lua:71]: Great!

**Comment by kyazdani42** [comment]
thanks ;) but i got an error when doing that on the first line (the `..` thing)

**Pull request merged**